fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/RdfLexicon.java

```diff
@@ -81,25 +76,19 @@
      */
     public static final String LDP_NAMESPACE = "http://www.w3.org/ns/ldp#";
 
-    /**
-     * SPARQL service description namespace.
-     */
-    public static final String SPARQL_SD_NAMESPACE =
-            "http://www.w3.org/ns/sparql-service-description#";
-
     /**
      * Is this namespace one that the repository manages?
      */
     public static final Predicate<String> isManagedNamespace = p -> p.equals(REPOSITORY_NAMESPACE) ||
             p.equals(LDP_NAMESPACE);
 
     // MEMBERSHIP
-    public static final Property HAS_PARENT =
+    private static final Property HAS_PARENT =
```

Comment:
I don't think HAS_PARENT is ever populated anymore (I believe we deleted the class that added it a few weeks ago), so it seems like we can remove this property and update `membershipProperties`

Reply:
I think `HAS_CHILD` can also be safely removed. We only use it in tests along the lines of: "assert HAS_CHILD does not exist".